python_scripts/linear_models_sol_05.py

```diff
@@ -108,6 +108,9 @@ def plot_decision_function(fitted_classifier, range_features, ax=None):
         hue=target_column, palette=["tab:red", "tab:blue"])
     plot_decision_function(logistic_regression, range_features, ax=ax)
     plt.title(f"C: {C}")
+    plt.show()
```

Comment:
We can use a cell for the plot and a cell to compute the score
```suggestion
    
    # %%
```

Reply:
The score is computed inside a for loop that starts in line 101. The goal was to score different values for `C` on the fly.

Reply:
Nevermind, I will add a legend to each plot containing the score as suggested below. Thanks!